### ISSUE TITLE
Ship with a default todo.txt file

### ISSUE THREAD

**Issue by smichel17** [mpcjanssen/simpletask-android#911]
The tasks in it would be things like:

- Create a new task
- Change the filter
- Open the right drawer and save the filter
- Look through the settings
etc

Originally @wolftune's idea, it would basically be a tutorial that people would walk through and get a feel for using the app when it's first installed.

**Comment by sefernet** [comment]
+1
Seems helpfull

**Comment by smichel17** [comment]
I don't think this would be hard to implement; the main work would be putting together a coherent list of tasks to start out with, that covers all the most important hard-to-find features without being too long/annoying

**Comment by smichel17** [comment]
Things that I know are not intuitive / discoverable (or at least, that people have not found in the past):

- Long press on task to open links
- Swipe in from the right to open the saved filter drawer
- Steps to archive all tasks in the current view (select task, select all, archive)

Probably you could find more by searching this bug tracker for "discoverable"

**Comment by aisbergde** [comment]
Great idea, I also missed some these features, now I know.
To provide a default todo list is one way, a menu point "how to start" ore "Quick start" would also helpful. Intuitively I would search in Settings > help, And there should be one very present menu point. Or it could be a separat entry in the apps menu.

But in General the idea to just deliver a first todo.txt explaining idea and usage is a very good way. In the apps menu could be implemented an "open sample file".

**Comment by smichel17** [comment]
Well, the app already has a default todo.txt file, so there's no extra menu needed. Just (pseudocode):

```
when (app is first opened):
  if (default todo.txt file is empty or does not exist):
    insert default tasks into default todo.txt file
```

The check that the file is empty is important, because we wouldn't want to add a bunch of tasks to someone's existing file when they reinstall the app.

If someone wants to compile a good list of introductory tasks, I'll implement it.

**Comment by sphenotux** [comment]
Volunteering.
